models/interview.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+const ObjectId = Schema.Types.ObjectId;
+
+/**
+ * Interview Model
+ */
+module.exports = mongoose.model(
+  "Interview",
+  new Schema({
+    user: {
+      type: ObjectId,
+      required: true,
+    },
+    ambassador: {
+      type: ObjectId,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    phone: {
+      type: String,
+      required: true,
+    },
+    date: {
+      type: String,
+      required: true,
+    },
+    time: {
+      type: String,
+      required: true,
+    },
+    location: {
+      type: String,
+      required: true,
+    },
+    internalNotes: {
+      type: String,
+      required: true,
```

Comment:
@dnanjunda Do we want `internalNotes` to be an optional field here as well?

Reply:
Sounds good :+1: fixed!